shared/main/java/net/ccbluex/liquidbounce/features/module/modules/movement/speeds/other/Mineplex.kt

```diff
@@ -0,0 +1,51 @@
+/*
+ * LiquidBounce Hacked Client
+ * A free open source mixin-based injection hacked client for Minecraft using Minecraft Forge.
+ * https://github.com/CCBlueX/LiquidBounce/
+ */
+package net.ccbluex.liquidbounce.features.module.modules.movement.speeds.other
+
+import net.ccbluex.liquidbounce.LiquidBounce
+import net.ccbluex.liquidbounce.event.MoveEvent
+import net.ccbluex.liquidbounce.features.module.modules.movement.Speed
+import net.ccbluex.liquidbounce.features.module.modules.movement.speeds.SpeedMode
+import net.ccbluex.liquidbounce.utils.MovementUtils
+
+class Mineplex : SpeedMode("Mineplex") {
+    
+    val speed1 = 0
```

Comment:
The variables has to be mutable. Please learn Kotlin and use a IDE like Jetbrains IntelliJ to check if the code is fully working.